### PR TITLE
Docs for variant.booking_fee_rate

### PR DESCRIPTION
**Pull request by ianmooney**
The `booking_fee_rate` was introduced in this PR https://github.com/easolhq/easol/pull/7661

### CODE REVIEW

**Comment by ianmooney** [comment]
[THEMEDEVREQ: Expose booking fee rate on variants](https://trello.com/c/bEzoOB3A/340-themedevreq-expose-booking-fee-rate-on-variants)

File: docs/reference/objects/product/variant/index.md [hunk 1]
```diff
@@ -19,6 +19,13 @@ object
 
 #### Attributes
 
+## `variant.booking_fee_rate`
+{: .d-inline-block }
+number
+{: .label .fs-1 }
+
```
Comment (geoffreymm) [inline docs/reference/objects/product/variant/index.md:26]: Should we mention that this fallback to the company booking fee rate if not present?

**Review by geoffreymm — APPROVED** [review]

File: docs/reference/objects/product/variant/index.md (hunk 1 above)
Comment (ianmooney) [reply docs/reference/objects/product/variant/index.md:26]: I don't think that needs to be mentioned here to be honest.

Comment (geoffreymm) [reply docs/reference/objects/product/variant/index.md:26]: Maybe you right. I was thinking that if no fee is defined on the variant, one might think that this return nil or something, but from a theme developer that might be irrelevant

**Pull request merged**